flake.nix

```diff
@@ -78,7 +78,18 @@
             buildInputs = with pkgs; [
               ccache
               ninja
+              llvmPackages_19.clang-tools
             ];
+
+            cmakeFlags = self.packages.${system}.prismlauncher-unwrapped.cmakeFlags ++ [
+              "-GNinja"
+              "-Bbuild"
+            ];
+
+            shellHook = ''
+              cmake $cmakeFlags -D CMAKE_BUILD_TYPE=Debug
+              cp {build/,}compile_commands.json
```

Comment:
it's more common to syn-link it so it autoupdates during development. any particular reason you don't here?

Reply:
I was going to do that, but I was concerned about it ended up dangling if you deleted the build directory -- which I happen to do pretty frequently

This may break if you made some larger changes though, so it probably would be better. Plus you can always run `cmake` yourself ig